Tribler/Core/Modules/restapi/my_channel_endpoint.py

```diff
@@ -113,6 +118,52 @@ def render_PUT(self, request):
         return json.dumps({"added": channel_id})
 
 
+class MyChannelCreateTorrentEndpoint(MyChannelBaseEndpoint):
+    """
+    Create a torrent file from local files.
+
+    Example request:
+    {
+        "files": ['path/to/file.txt', 'path/to/another/file.mp4'],
+        "tracker": "http://localhost/announce",
+        "name": "Cat video",
+        "description: "Video of my cat"
```

Comment:
In the GUI, we have some more options, see https://gyazo.com/1fc20510d30ec13b06a8371605c62d6c. Do we want to add these?